src/index.js

```diff
@@ -426,7 +426,7 @@ async function facebookMain(
    * Waiting for the group stories container to continue
    * and to avoid the selector not found error
   **/
-  await page.waitForXPath('//*[@id="m_group_stories_container"]');
+  await page.waitForXPath('//div[@id="content_container"]');
```

Comment:
Note that this is not valid on the new Facebook design:

![image](https://user-images.githubusercontent.com/42304709/119815565-99ac6d80-bee3-11eb-82b6-a4d7b9a0e126.png)

A new selector is needed here, posts usually are inside a `div` element with the role `feed`:

![image](https://user-images.githubusercontent.com/42304709/119815881-f4de6000-bee3-11eb-83f5-4668120b49ad.png)

In fact, not all of these elements are submissions, only elements with the selector `div.du4w35lb.k4urcfbm.l9j0dhe7.sjgh65i0` are.

What I'm not sure about is if Facebook usually changes these class names when updating their website, this would be an issue if it is true.

Reply:
Hi @iMrDJAi ,

You're completely right. Also, for the class names, Facebook renders using a generated JavaScript so class names are probably  auto-generated

Reply:
> Hi @iMrDJAi ,
> 
> You're completely right. Also, for the class names, Facebook renders using a generated JavaScript so class names are probably auto-generated

@kaanyagci Yeah, this seems to be challenging.